publications-2018.bib

```diff
@@ -723,6 +723,29 @@ @article{GKK18
    doi     = {}
 }
 
+@PhdThesis{SergeyCharnyiPhd2018,
+  Title                    = {The {EMAC} scheme for Navier-Stokes simulations, and application to flow past bluff bodies},
+  Author                   = {Sergey Charnyi},
+  School                   = {Clemson University},
+  Year                     = {2018},
+  Url                      = {https://tigerprints.clemson.edu/all_dissertations/2243/}
+}
+
+@PhdThesis{LiangZhaoPhd2018,
+  Title                    = {A numerical method for the Navier-Stokes equations in velocity-vorticity form},
```

Comment:
I am not sure if it matters, but the thesis title did not have a hyphen between Navier and Stokes.

Reply:
Sigh.

Reply:
I think that we need to decide how pedantic we're going to be about these things.

Reply:
I cannot imagine a circumstance where the hyphen would matter. Lets not bother with fixing it.

Reply:
No, it makes sense. I was just annoyed that his title should have a hyphen. Thanks guys!